### ISSUE TITLE
Loss of focus is observed after tabbing from "No, I would not to participate" radio button

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#374]
### Steps to Reproduce
1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Navigate to Tools -> Preferences and invoke it. This will invoke general tab of preferences window.
3. Invoke Feedback tab using Mouse (As it is not accessible by keyboard).
4. Navigate to "No, I would not to participate" radio button and press tab.

### Expected Behavior
Keyboard focus should move to next interactive element (Feedback Tab).
### Actual Behavior
1)Loss of focus is observed after pressing tab from "No, I would not to participate" radio button. 
2) If user press the tab(2nd tab) again it sets focus on "Yes, I would participate (Recommended)" radio button.

Inspect Behavior:
After pressing tab from "No, I would not to participate" radio button, focus is going to content area ("Help Improve Xamarin Workbooks") of preferences. Please refer below screenshot.
![screenshot 175](https://user-images.githubusercontent.com/22956130/40759943-fc659814-64b1-11e8-9b99-f9743ca85207.png)
### User Impact:
People with visual impairments can become disoriented when keyboard navigation takes focus somewhere unexpected or when they cannot easily find the content surrounding an interactive element.

### Screenshots:
![issue 8](https://user-images.githubusercontent.com/22956130/40759991-40a0413c-64b2-11e8-99ab-b12cb2327887.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS2.4.3

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
